products/compute/terraform.yaml

```diff
@@ -1959,14 +1959,17 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "ssl_certificate_basic"
         primary_resource_id: "default"
+        skip_vcr: true
```

Comment:
Can we get some more comments about why specific tests are being skipped? Doesn't have to be super long, just something like "randomness" or "fine-grained order"